### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by anq420**

### CODE REVIEW

File: lesson_12_hm.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        if self.status == 'not started':
+            print('I am waiting to start engine to start moving')
+        if self.status == 'started':
+            print('I am moving')
+
+    def launch(self):
+        if self.status is None:
+            self.status = 'started'
+            print(self.status)
+        elif self.status is not None:
```
Comment (nicilytop) [inline lesson_12_hm.py]: lets connect with Bogdan.. the same error

Comment (nicilytop) [reply lesson_12_hm.py]: we dont need this check.. if my car was muted and I will want mute it - I do it again - it is amazing

**Review by nicilytop — CHANGES_REQUESTED** [review]
correct you check

**Review by nicilytop-tms — APPROVED** [review]
good job 😎

**Pull request closed without merging**